falco/values.yaml

```diff
@@ -342,3 +342,47 @@ tolerations:
 scc:
   # true here enabled creation of Security Context Constraints in Openshift
   create: true
+
+# Add initContainers to falco pod
+extraInitContainers: []
+
+# Below, there is an example initContainer that compiles the eBPF probe from downloaded kernel sources.
+# You need to change the falco image to use falco-no-driver
+# and modify the daemonset.env section to add the following env variable
+# FALCO_BPF_PROBE: /host/lib/modules/falco-bpf.o
+# 
+# replacement of falcosecurity/falco image with
+# falcosecurity/falco-no-driver is also required, otherwise falco will try to
+# download and compile the driver at startup time.
+
+# extraInitContainers:
+#   - name: init-driverloader
+#     image: falcosecurity/falco-driver-loader:0.25.0
+#     env:
+#       - name: FALCO_BPF_PROBE
+#       - name: BPF_USE_LOCAL_KERNEL_SOURCES
+#         value: "true"
+#       - NAME: ENABLE_DOWNLOAD
+#         value: ""
+#     command: ["/bin/bash"]
+#     args: ["-c", "apt update; apt install -f -y; apt install -y flex bison; apt install -y gcc libssl-dev; falco-driver-loader; cp /root/.falco/falco-bpf.o /host/lib/modules/falco-bpf.o"] 
```

Comment:
Ok, I have found some time to try it.

Although the overall PR seems good to me, unfortunately, I found some issue here that needs to be addressed:

- the `falcosecurity/falco-driver-loader` should not need `apt update; apt install -f -y; apt install -y flex bison; apt install -y gcc libssl-dev;` to properly compile the driver, and that let me think there's a problem in that image
- I don't believe we should encourage folks to put the eBPF probe in `/host/lib/modules/falco-bpf.o`, but I understand we need a place to put it so the main container can use it

Thus, I just need to put this on hold until we found a solution, especially for the latter point.

However, it's not your fault! Thanks to you we have noticed these other issues! :+1: